src/main/java/core/basesyntax/products/Goods.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.products;
+
+import core.basesyntax.constants.Product;
+import core.basesyntax.products.impl.Apple;
+import core.basesyntax.products.impl.Banana;
+import java.util.Collections;
+import java.util.Map;
+
+public interface Goods {
+    public static final Map<Product, Goods> productsMap = Collections.unmodifiableMap(
+            Map.of(Product.APPLE, new Apple(), Product.BANANA, new Banana()));
```

Comment:
Why you don`t want to use one map(prodType, quantity) instead of Map and List?

Reply:
i should use map in database?

Reply:
Oleh requested you in the very first comment "Handle fruits as just strings to simplify the code".
And not only simplicity.
Now if report contains oranges - then that's it. App not working. Need to create model. Then potato arrives, again new model and code changes. And same for each new position.
That's not the way app should be maintained.

The simplest way is to use Map<String, Integer> as storage (just pairs name/quantity).
Possibly also some `List<Product>` but when Product is some class with fields "name" and "quantity", not interface. But it would be much harder to do.

Anyway I wish to see solution that will not crash if we decided to add new position without that IT guy